zebrad/tests/acceptance.rs

```diff
@@ -321,6 +321,7 @@ where
         args.extend_from_slice(extra_args);
 
         self.spawn_child_with_command("lightwalletd", &args)
+            .map(|child| child.bypass_test_capture(true))
```

Comment:
Flexibility: this always prints the output, regardless of the test settings. But we might want to have the output in some tests, but hide it in others.

Can we add `--nocapture` to the CI job instead?

Reply:
Done.